dev/src/ion2-calendar/services/calendar.service.ts

```diff
@@ -12,10 +12,15 @@ import {
 import { defaults, pickModes } from '../config';
 
 const isBoolean = (input: any) => input === true || input === false;
+import { DEFAULT_CALENDAR_OPTIONS } from './calendar-options.provider';
```

Comment:
Looks like the import here is out of place

Reply:
Ah! Didn't see that. Good catch!